src/strings/PowerFxResources.en-US.resx

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
>tandalone="yes"? [](http://example.com/codeflow?start=38&length=16)

Is this intentional change?

Reply:
Absolutely not :)
I don't know what "standalone" adds here but i'll have it back

Reply:
fwiw - I'm seeing this is in a bunch of PRs ... some VS thing is automatically doing this.